pkg/storegateway/series_chunks_test.go

```diff
@@ -574,6 +589,40 @@ func TestLoadingSeriesChunksSetIterator(t *testing.T) {
 			loadErr:      errors.New("test err"),
 			expectedErr:  "test err",
 		},
+		"loads single set from single block with some ignored histogram chunks": {
```

Comment:
Can you please add a test case where all the chunks for a series are native histograms? You can also try with cases where all series have only native histograms, and the `loadingSeriesChunksSetIterator` returns a single empty batch. My understanding is that are realistic scenarios

Reply:
Yep, absolutely!